Source/NetworkReachabilityManager.swift

```diff
@@ -124,12 +124,14 @@ public class NetworkReachabilityManager {
     /// - returns: The new `NetworkReachabilityManager` instance.
     public convenience init?() {
         var address = sockaddr_in()
-        address.sin_len = UInt8(sizeofValue(address))
+        address.sin_len = UInt8(MemoryLayout<sockaddr_in>.size)
         address.sin_family = sa_family_t(AF_INET)
 
-        guard let reachability = withUnsafePointer(&address, {
-            SCNetworkReachabilityCreateWithAddress(nil, UnsafePointer($0))
-        }) else { return nil }
+        guard let reachability = withUnsafePointer(to: &address, ({ pointer in
+            return pointer.withMemoryRebound(to: sockaddr.self, capacity: MemoryLayout<sockaddr>.size) {
+                return SCNetworkReachabilityCreateWithAddress(nil, $0)
+            }
+        })) else { return nil }
```

Comment:
Fixed in [a4a81](https://github.com/Alamofire/Alamofire/pull/1413/commits/356204b0711893f898caf042d093e1f7344a4a81).